File: practicas/src/practica3/services/impl/WholesaleBuyer.java
package practica3.services.impl;

import practica3.services.WelcomeUser;

import javax.swing.*;

public class WholesaleBuyer implements WelcomeUser {

    @Override
    public void welcomeUser(String name) {
        JOptionPane.showConfirmDialog(null, name+" We have send you and email ;Do you accept the terms and conditions?");
    }
}

File: ParcialAnalisis/Services/Imp/User/WholeSaleBuyer.py
from Services.userServices.UserType import UserType

'''
This class represents an user type
'''
class WholeSaleBuyer(UserType):

    def __init__(self):
        super().__init__()

    def sayHello(self):
        print("We have send you and email ;Do you accept the terms and conditions?")